### PR TITLE
Quick 2024 update

### PR DESCRIPTION
**Pull request by tim-schilling**
- Switches the domain from 2023.djangocon.us to 2024.djangocon.us
- Updates the config for 2024's dates
- Updates the temporary landing page with 2024's dates
- Updates the invitation letter for 2024 and Peter as DEFNA president.

### CODE REVIEW

**Comment by tim-schilling** [comment]
We still need to add a link to the VISA related pages on the landing page and get the vercel app wired up.

File: .github/settings.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 repository:
   description: 🫐 The DjangoCon US 2023 conference website
```
Comment (PeterGrand) [inline .github/settings.yml]:
```suggestion
  description: 🫐 The DjangoCon US 2024 conference website
```

File: _config.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@ title: DjangoCon US
 url: "https://2024.djangocon.us"
 author : DjangoCon US
 description: |
-  DjangoCon US 2024 conference will take place in Fall, 2024.
+  DjangoCon US 2024 conference will take place in Durham, NC
+  from September 22-27, 2023!
```
Comment (PeterGrand) [inline _config.yml]:
```suggestion
  from September 22-27, 2024!
```

File: _config.yml [hunk 3]
```diff
@@ -109,14 +110,14 @@ sponsors_email: "sponsors@djangocon.us"
 visa_email: "visas@djangocon.us"
 
 # Our DjangoCon US links
-cfp_application: "https://pretalx.com/djangocon-us-2023/cfp"
+cfp_application: "https://pretalx.com/djangocon-us-2024/cfp"
 hotel_reservation_link: "https://www.marriott.com/en-us/hotels/rducv-durham-marriott-city-center/overview/"
 # slack_link: "https://djangoconus2022.herokuapp.com"
 merchandise_link: "https://django.threadless.com/designs/djangocon-us-2023/mens/t-shirt/regular?variation=front&color=lemon"
```
Comment (PeterGrand) [inline _config.yml]:
```suggestion
hotel_reservation_link: ""
# slack_link: "https://djangoconus2022.herokuapp.com"
merchandise_link: ""
```

File: _drafts/2023-03-01-announcing-djangocon-us-2023.md [hunk 4]
```diff
@@ -20,15 +20,15 @@ The conference will have tutorials on Sunday, October 8, talks from Monday, Octo
 - October 16, 17, &amp; 18: Talks
 - October 19 &amp; 20: Sprints
 
-This year our venue will be [Durham Convention Center](https://2023.djangocon.us/venue/) in Durham, NC.
+This year our venue will be [Durham Convention Center](https://2024.djangocon.us/venue/) in Durham, NC.
 Tickets are [on sale now](https://ti.to/defna/djangocon-us-2023){% comment %}Tickets will be on sale soon{% endcomment %}, and our [hotel block is open](https://www.marriott.com/events/start.mi?id=1674740649947&key=GRP).
 
 Due to the combination of a Duke home football game on October 14 and another tech conference right after ours, hotel space will be *extremely* limited in 2023.
 {% comment %}We sold out in previous years, so please take advantage of our early bird pricing!{% endcomment %}
```
Comment (PeterGrand) [inline _drafts/2023-03-01-announcing-djangocon-us-2023.md]:
```suggestion
{% comment %}Tickets are [on sale now](https://ti.to/defna/djangocon-us-2023){% endcomment %}Tickets will be on sale soon, and our hotel block will be announced shortly.

We sold out in previous years, so please take advantage of our early bird pricing!
```

File: _posts/2023-09-09-final-hotel-call.md [hunk 5]
```diff
@@ -6,16 +6,16 @@ layout: post
...
-that can be attended from the comfort of your own home. Perhaps you're ready to dive into the world of [Postgres optimization](https://2023.djangocon.us/tutorials/seamless-postgres-query-optimization/). Or maybe you're more into [Airflow](https://2023.djangocon.us/tutorials/django-3-airflow/) or [Wagtail](https://2023.djangocon.us/tutorials/best-of-both-worlds-next-js-wagtail/).
-Give the [whole collection](https://2023.djangocon.us/tutorials/) a look and purchase a ticket.
+fantastic conference this fall. We have amazing keynotes and talks lined up. And [there are tutorials](https://2024.djangocon.us/schedule/#Day-Tutorials) the weekend before
+that can be attended from the comfort of your own home. Perhaps you're ready to dive into the world of [Postgres optimization](https://2024.djangocon.us/tutorials/seamless-postgres-query-optimization/). Or maybe you're more into [Airflow](https://2024.djangocon.us/tutorials/django-3-airflow/) or [Wagtail](https://2024.djangocon.us/tutorials/best-of-both-worlds-next-js-wagtail/).
```
Comment (PeterGrand) [inline _posts/2023-09-09-final-hotel-call.md]:
```suggestion
that will be available at the venue on September 22, 2024.
```

**Review by PeterGrand — COMMENTED** [review]
Left suggestions for a few places that needed updates

File: _drafts/2023-03-01-announcing-djangocon-us-2023.md [hunk 6]
```diff
@@ -20,15 +20,15 @@ The conference will have tutorials on Sunday, October 8, talks from Monday, Octo
 - October 16, 17, &amp; 18: Talks
```
Comment (drewbrew) [inline _drafts/2023-03-01-announcing-djangocon-us-2023.md:20]: These dates need updating too?

**Review by drewbrew — APPROVED** [review]

File: _drafts/2023-03-01-announcing-djangocon-us-2023.md (hunk 6 above)
Comment (tim-schilling) [reply _drafts/2023-03-01-announcing-djangocon-us-2023.md:20]: This is an old blog post. When we post the replacement, we can adjust it.

**Review by PeterGrand — APPROVED** [review]

**Comment by tim-schilling** [comment]
Header on landing page:
image

Header on blog post:
image

Footer on blog post:
image

**Review by PeterGrand — APPROVED** [review]

**Review by drewbrew — APPROVED** [review]

**Comment by jefftriplett** [comment]
@tim-schilling we might as well delete the old schedule (talks, tutorials, etc) and older blog posts to prevent confusion since they show up in the sitemaps.

**Comment by tim-schilling** [comment]
@jefftriplett done in #8

**Pull request merged**